### ISSUE TITLE
unresolved import from std

### ISSUE THREAD

**Issue by reison1218** [rust-analyzer/rust-analyzer#7470]
here is issue happen to me every day.
![1611815849870](https://user-images.githubusercontent.com/39828011/106099897-63015900-6176-11eb-84c3-f6c9b60f4bc3.jpg)

rust version:
![1611815994327](https://user-images.githubusercontent.com/39828011/106100101-bd021e80-6176-11eb-9b75-34845ca784ce.jpg)

rust-analyzer version:0.2.465

**Comment by bjorn3** [comment]
https://github.com/rust-lang/rust/blob/e32f372c4203b2527221b313cf63b05ea178e8a9/library/std/src/collections/hash/map.rs#L6

This line imports hashbrown, which is a dependency of libstd from crates.io. Rust-analyzer doesn't yet handle crates.io dependencies for the sysroot. You shouldn't see any error when you close that file.

**Comment by reison1218** [comment]
yeah,after i close the file,there is no error anymore!look forward to this plug more complete

**Comment by jyn514** [comment]
Is there a tracking issue for sysroot support? I've seen quite a few bugs related to it.

**Comment by Veykril** [comment]
We don't check non-workspace deps for diagnostics any longer
